components/expression/condition.jsx

```diff
@@ -0,0 +1,206 @@
+// shortid is a short, non-sequential, url-friendly, unique id generator
+import shortid from 'shortid';
+import { EXPRESSION_CONDITION } from '../../utilities/constants';
+
+import Combobox from '../combobox';
+import Input from '../input';
+import Button from '../button';
+
+const propTypes = {
+	assistiveText: PropTypes.shape({
+		title: PropTypes.string,
+		delete: PropTypes.string
+	}),
+	/**
+	 * HTML id for component.
+	 */
+	id: PropTypes.string,
+	/**
+	 * Visual Picker accepts `Checkbox`, `Radio` and `VisualPickerLink` components as children. Please see `Checkbox`, `Radio` and `VisualPickerLink` for props.
+	 */
+	children: PropTypes.node,
+	/**
+	 * CSS classes to be added to tag with `.slds-form-element`. Uses `classNames` [API](https://github.com/JedWatson/classnames).
+	 */
+	className: PropTypes.oneOfType([
+		PropTypes.array,
+		PropTypes.object,
+		PropTypes.string,
+	]),
+
+	labels: PropTypes.shape({
+		condition: PropTypes.string,
+		resource: PropTypes.string,
+		operator: PropTypes.string,
+		value: PropTypes.string
+	}),
+
+	resourcesList: PropTypes.arrayOf(PropTypes.object),
+	resourceSelected: PropTypes.object,
+	onChangeResource: PropTypes.func,
+
+	operator: PropTypes.oneOf(['equal', 'notEqual', 'greaterThan', 'lessThan']),
+
+	onChangeOperator: PropTypes.func,
+
+	onDelete: PropTypes.func
+
+};
+
+const operatorSelections = [
+	{
+		id: '1',
+		label: 'Equals',
+	},
+	{
+		id: '2',
+		label: 'Does Not Equals',
+	},
+	{
+		id: '3',
+		label: 'Greater Than',
+	},
+	{
+		id: '4',
+		label: 'Less Than',
+	}
+];
+
+/**
+ * Expression Condition Component
+ */
+class ExpressionCondition extends React.Component {
+	componentWillMount() {
+		this.generatedId = shortid.generate();
+	}
+
+	/**
+	 * Get the Expression Condition's HTML id. Generate a new one if no ID present.
+	 */
+	getId() {
+		return this.props.id || this.generatedId;
+	}
+
+	getResourceSelected(){
+		const selection = this.props.resourceSelected;
+		const selected = [];
+		selected.push(selection);
+		return selected;
+	}
+
+	getOperatorSelected(){
+		const selected = [];
+		const operator = this.props.operator;
+		if(operator === "equal"){
+			selected.push(operatorSelections[0])
+		} else if(operator === "notEqual"){
+			selected.push(operatorSelections[1])
+		} else if(operator === "greaterThan"){
+			selected.push(operatorSelections[2])
+		} else if(operator === "lessThan"){
+			selected.push(operatorSelections[3])
+		}
+		return selected;
+	}
+
+	operatorChange(event, data)
+	{
+		const selection = data.selection[0].id;
+		let operator = "";
+		if(selection === '1') {
+			operator = "equal"
+		} else if(selection === '2'){
+			operator = "notEqual"
+		} else if(selection === '3'){
+			operator = "greaterThan"
+		} else if(selection === '4'){
+			operator = "lessThan"
+		}
+		// eslint-disable-next-line no-unused-expressions
+		typeof this.props.onChangeOperator === "function" ? this.props.onChangeOperator(operator) : null;
```

Comment:
Please remove the `eslint-disable-next-line` comment, then convert this to:
```
if (this.props.onChangeOperator) {
    this.props.onChangeOperator(operator);
}
```
PropTypes already protects the expected function type, so we just want to make sure it's there before running the function.